tests/playgrounds/advanced_playground.html

```diff
@@ -3,29 +3,33 @@
 <head>
 <meta charset="utf-8">
 <title>Advanced Blockly Playground</title>
-<script src="../../blockly_uncompressed.js"></script>
-<script src="../../msg/messages.js"></script>
-<script src="../themes/test_themes.js"></script>
+<script src="prepare.js"></script>
 <script src="./screenshot.js"></script>
+<script src="../themes/test_themes.js"></script>
 <script src="../../node_modules/@blockly/dev-tools/dist/index.js"></script>
 <script src="../../node_modules/@blockly/theme-modern/dist/index.js"></script>
 
 <script>
-// Custom requires for the playground.
-// Rendering.
-goog.require('Blockly.minimalist.Renderer');
-goog.require('Blockly.Themes.Zelos');
-
-// Other.
-goog.require('Blockly.WorkspaceCommentSvg');
-goog.require('Blockly.libraryBlocks');
-goog.require('Blockly.Dart.all');
-goog.require('Blockly.JavaScript.all');
-goog.require('Blockly.Lua.all');
-goog.require('Blockly.PHP.all');
-goog.require('Blockly.Python.all');
+const IS_UNCOMPRESSED = Boolean(window.BlocklyLoader);  // See prepare.js
+
+if (IS_UNCOMPRESSED) {
+  // Custom requires for the playground.
+  // Rendering.
+  goog.require('Blockly.minimalist.Renderer');
+  goog.require('Blockly.Themes.Zelos');
+
+  // Other.
+  goog.require('Blockly.WorkspaceCommentSvg');
+  goog.require('Blockly.libraryBlocks');
+  goog.require('Blockly.Dart.all');
+  goog.require('Blockly.JavaScript.all');
+  goog.require('Blockly.Lua.all');
+  goog.require('Blockly.PHP.all');
+  goog.require('Blockly.Python.all');
```

Comment:
I think these should be bundled in by `goog.bootstrap` in prepare.js, could you check if these can be removed?